code/_globalvars/configuration.dm

```diff
@@ -4,7 +4,7 @@ var/datum/protected_configuration/protected_config = null
 var/host = null
 var/join_motd = null
 var/station_name = null
-var/game_version = "/tg/ Station 13"
+var/game_version = "CP Station 13"

```

Comment:
Those are not mods, ;) these are mods. http://steamcommunity.com/id/gartley/myworkshopfiles/